argopy/xarray.py

```diff
@@ -656,3 +663,108 @@ def interp_std_levels(self, std_lev):
         ds_out.argo._add_history('Interpolated on standard levels')
         
         return ds_out
+
+    def teos10(self, vlist: list = ['SA', 'CT', 'SIG0', 'N2', 'PV', 'PTEMP']):
```

Comment:
Just a thought, what about an "inplace" option ? in pyxpcm I like the idea of choosing to add new var to existing dataset or building a new one.

Reply:
Good idea, you're right
In the present form, it is operating "in place"
Working on this !